<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialVar_Matrix</class>
 <widget class="QWidget" name="MaterialVar_Matrix">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="3">
       <widget class="QDoubleSpinBox" name="txt_3_2">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.999999046325684</double>
        </property>
        <property name="maximum">
         <double>999999999.999999046325684</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDoubleSpinBox" name="txt_2_0">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.999999046325684</double>
        </property>
        <property name="maximum">
         <double>999999999.999999046325684</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="txt_1_0">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.999999046325684</double>
        </property>
        <property name="maximum">
         <double>999999999.999999046325684</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QDoubleSpinBox" name="txt_0_1">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.999999046325684</double>
        </property>
        <property name="maximum">
         <double>999999999.999999046325684</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QDoubleSpinBox" name="txt_0_0">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.999999046325684</double>
        </property>
        <property name="maximum">
         <double>999999999.999999046325684</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QDoubleSpinBox" name="txt_3_3">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.999999046325684</double>
        </property>
        <property name="maximum">
         <double>999999999.999999046325684</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="txt_3_0">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.999999046325684</double>
        </property>
        <property name="maximum">
         <double>999999999.999999046325684</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="txt_1_1">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.999999046325684</double>
        </property>
        <property name="maximum">
         <double>999999999.999999046325684</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QDoubleSpinBox" name="txt_0_2">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.999999046325684</double>
        </property>
        <property name="maximum">
         <double>999999999.999999046325684</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="txt_1_3">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.999999046325684</double>
        </property>
        <property name="maximum">
         <double>999999999.999999046325684</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QDoubleSpinBox" name="txt_0_3">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.999999046325684</double>
        </property>
        <property name="maximum">
         <double>999999999.999999046325684</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="txt_1_2">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.999999046325684</double>
        </property>
        <property name="maximum">
         <double>999999999.999999046325684</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="txt_3_1">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.999999046325684</double>
        </property>
        <property name="maximum">
         <double>999999999.999999046325684</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="txt_2_3">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.999999046325684</double>
        </property>
        <property name="maximum">
         <double>999999999.999999046325684</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="txt_2_1">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.999999046325684</double>
        </property>
        <property name="maximum">
         <double>999999999.999999046325684</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="txt_2_2">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>-999999999.999999046325684</double>
        </property>
        <property name="maximum">
         <double>999999999.999999046325684</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txt_0_0</tabstop>
  <tabstop>txt_1_0</tabstop>
  <tabstop>txt_2_0</tabstop>
  <tabstop>txt_3_0</tabstop>
  <tabstop>txt_0_1</tabstop>
  <tabstop>txt_1_1</tabstop>
  <tabstop>txt_2_1</tabstop>
  <tabstop>txt_3_1</tabstop>
  <tabstop>txt_0_2</tabstop>
  <tabstop>txt_1_2</tabstop>
  <tabstop>txt_2_2</tabstop>
  <tabstop>txt_3_2</tabstop>
  <tabstop>txt_0_3</tabstop>
  <tabstop>txt_1_3</tabstop>
  <tabstop>txt_2_3</tabstop>
  <tabstop>txt_3_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
